libs/common/Memory/LimitedFixedBufferPool.cs

```diff
@@ -98,6 +138,20 @@ public unsafe PoolEntry Get(int size)
             return new PoolEntry(size, this);
         }
 
+        /// <summary>
+        /// Purge pool entries from all levels
+        /// </summary>
+        public void Purge()
+        {
+            for (var i = 0; i < numLevels; i++)
+            {
+                if (pool[i] == null) continue;
+                // Keep trying Dequeuing until no items left to free
+                while (pool[i].items.TryDequeue(out var _))
```

Comment:
add comment to clarify the intended difference between Purge and Dispose's wrt removing items